docroot/modules/custom/va_gov_backend/src/EventSubscriber/EntityEventSubscriber.php

```diff
@@ -96,15 +119,114 @@ public function addStateManagementToBioFieldsSubmitHandler(FormIdAlterEvent $eve
+   * Locks down form fields for non-admins.
+   *
+   * @param array $form
+   *   The form.
+   */
+  public function lockCentralizedContentFields(array &$form) {
+    // All users should see this intro.
+    $form['title']['#prefix'] = '<h2>' . $this->t('About') . '</h2><p>' . $this->t("Centralized content ensures that Veterans receive the same information about certain topics no matter where they're presented on VA.gov. Here you'll mangage content components that contain pieces of standardized information. Review the about info below to understand where this content will be applied.") . '</p>';
+    $form['title']['#attributes'] = [
+      'class' => ['cc-special-treatment-field'],
+    ];
+    $form['field_content_block']['#suffix'] = '<div class="cc-suffix-text"><strong>' . $this->t('Need to add a new centralized content component not listed here?') . '</strong><br />' . $this->t('Contact the product owner associated with this content.') . '</div>';
+
+    if (!$this->userPermsService->hasAdminRole(TRUE)) {
+      $form['#attached']['library'][] = 'va_gov_backend/centralized_content_alterations';
+
+      // For non-admin replace field with static content.
+      unset($form['field_content_block']['widget']['#title']);
+      $form['field_content_block']['widget']['#title'] = '<h2>' . $this->t('Content') . '</h2>';
+
+      // For non-admin replace body field with static content
+      // generated from field value.
+      if (!empty($form['body']['widget'][0]['#default_value'])) {
+        $form['body'] = [
+          '#markup' => '<h3>' . $this->t('Purpose') . '</h3><div class="cc-wysi-wrap">' . $form['body']['widget'][0]['#default_value'] . '</div>',
+        ];
+      }
+      else {
+        unset($form['body']);
+      }
+
+      // For non-admin replace product field with static content
+      // generated from field value and disable editing.
+      if (!empty($form['field_product']['widget']['#default_value'][0])) {
+        $form['field_product'] = [
+          '#prefix' => '<h3>' . $this->t(':title', [':title' => $form['field_product']['widget']['#title']]) . '</h3><p class="cc-p">' . $this->t(':description', [':description' => $form['field_product']['widget']['#options'][$form['field_product']['widget']['#default_value'][0]]]) . '</p>',
+          '#attributes' => [
+            'class' => ['cc-special-treatment-field'],
+          ],
+        ];
+      }
+      else {
+        unset($form['field_product']);
+      }
+
+      // For non-admin replace applied to field with static content
+      // generated from field value and disable editing.
+      if (!empty($form['field_applied_to']['widget'][0]['value']['#default_value'])) {
+        $form['field_applied_to'] = [
+          '#prefix' => '<h3>' . $this->t(':title', [':title' => $form['field_applied_to']['widget'][0]['#title']]) . '</h3><p class="cc-p">' . $this->t(':description', [':description' => $form['field_applied_to']['widget'][0]['value']['#default_value']]) . '</p>',
```

Comment:
Moderate: also long and hard to see what it getting output.  How about?
```suggestion
        $title = $this->t(':title', [':title' => $form['field_applied_to']['widget'][0]['#title']]);
        $description = $this->t(':description', [':description' => $form['field_applied_to']['widget'][0]['value']['#default_value']]);
        $form['field_applied_to'] = [
          '#prefix' => "<h3>{$title"</h3><p class=\"cc-p\">$description</p>",
```